source/appendix_4.rst

```diff
@@ -93,3 +88,13 @@ configuration and a working WireGuard connection.
 .. raw:: latex
 
     \scalefont{1.25}
+
+.. _hosting-shared-variables-on-my-computer:
+
+---------------------------------------
+Hosting Shared Variables on My Computer
```

Comment:
What about renaming this section to "Hosting Network Shared Variables" and then in the table note saying "See the _Hosting Network Shared Varables_ section below." - just so that it is less ambiguous.